libs/test/SalesforceSDKTest/src/com/salesforce/androidsdk/rest/RestClientTest.java

```diff
@@ -420,15 +500,73 @@ public void testDelete() throws Exception {
         IdName newAccountIdName = createAccount();
 
         // Delete
-        RestResponse deleteResponse = restClient.sendSync(RestRequest.getRequestForDelete(TestCredentials.API_VERSION, "account", newAccountIdName.id));
+        RestResponse deleteResponse = restClient.sendSync(RestRequest.getRequestForDelete(TestCredentials.API_VERSION, ACCOUNT, newAccountIdName.id));
         assertTrue("Delete failed", deleteResponse.isSuccess());
 
         // Retrieve - expect 404
-        List<String> fields = Arrays.asList(new String[] {"name"});
-        RestResponse response = restClient.sendSync(RestRequest.getRequestForRetrieve(TestCredentials.API_VERSION, "account", newAccountIdName.id, fields));
+        List<String> fields = Arrays.asList(NAME);
+        RestResponse response = restClient.sendSync(RestRequest.getRequestForRetrieve(TestCredentials.API_VERSION, ACCOUNT, newAccountIdName.id, fields));
         assertEquals("404 was expected", HttpURLConnection.HTTP_NOT_FOUND, response.getStatusCode());
     }
 
+    /**
+     *
+     * if-unmodified-since not supported for delete
+     * Bring this test back once it is
+     *
+     * Testing delete calls to the server with if-unmodified-since.
+     * Create new account
+     * Update it
+     * then delete it with created date for unmodified since date (should not delete)
+     * then delete it with last modified date unmodified since date (should delete)
+     * @throws Exception
+     */
+    /*
+    public void testDeleteWithIfUnmodifiedSince() throws Exception {
```

Comment:
I wrote the test before I knew there was a server limitation. When run, it gives a 500 with a message saying "Please contact support" ;-)
We can remove the test entirely, and resuscitate it from github when the feature is available or leave it commented out. I don't have a strong preference either way.